src/m365/entra/commands/multitenant/multitenant-get.ts

```diff
@@ -0,0 +1,44 @@
+import { Logger } from '../../../../cli/Logger.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+
+interface MultitenantOrganization {
+  createdDateTime?: string;
+  displayName?: string;
+  description?: string;
+  id?: string;
+  state?: string;
+}
+
+class EntraMultitenantGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.MULTITENANT_GET;
+  }
+
+  public get description(): string {
+    return 'Gets properties of the multitenant organization';
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+
```

Comment:
We always aim to add at least 1 verbose message.

Reply:
ok